test/src/unit-capi-fragment_info.cc

```diff
@@ -671,6 +682,17 @@ TEST_CASE(
   rc = tiledb_fragment_info_get_fragment_name(ctx, fragment_info, 1, &name);
   CHECK(rc == TILEDB_OK);
 
+  tiledb_string_t* nameStr;
+  rc = tiledb_fragment_info_get_fragment_name_v2(
+      ctx, fragment_info, 1, &nameStr);
+  CHECK(rc == TILEDB_OK);
+  const char* nameStrData;
+  size_t nameStrLength;
+  rc = tiledb_string_view(nameStr, &nameStrData, &nameStrLength);
+  CHECK(rc == TILEDB_OK);
+  CHECK(std::string(name) == std::string(nameStrData));
```

Comment:
Same comment as above

On a quick glance it looks like there's a lot of copypasta here. This is a good time to eliminate it.